overlord/snapstate/snapstate.go

```diff
@@ -2347,18 +2347,20 @@ func ConfigDefaults(st *state.State, deviceCtx DeviceContext, snapName string) (
 		return nil, err
 	}
 
+	isSystemDefaults := snapName == defaultCoreSnapName
```

Comment:
// system configuration is kept under "core" so apply it's defaults when configuring "core"

Reply:
@bboozzoo  oops s/it's/its/